app/views/users/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<%= form_for @user do |f|%>
+  <%= f.label :username, "Username:"%>
+  <%= f.text_field :username, value: "u3" %><br><br>
+  <%= f.label :password, "Password:"%>
+  <%= f.password_field :password%><br><br>
+  <%= f.label :email, "Email:"%>
+  <%= f.text_field :email, placeholder: "example@example.com"%><br><br>
```

Comment:
This should be `f.email_field`.

Reply:
Thanks for the suggestion. It's been fixed